docs/proposals/version-documentation.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Documentation Versioning"
+linkTitle: "Documentation Versioning"
+weight: 1
+slug: documentation-versioning
+---
+
+- Author: [Jay Batra](https://github.com/jaybatra26)
+- Date: March 2020
+- Status: proposal
+
+## Problem
+
+In Cortex, currently, we are missing versioning of documentation. The idea is to have version documentation just like Prometheus.[`Prometheus`](https://prometheus.io/docs/introduction/overview/). Documentation is the main source of information for current contributors and first-timers. A properly versioned documentation will help everyone to have a proper place to look for answers before flagging it in the community. 
+
+In this proposal, we want to solve this. In particular, we want to:
+
+1. Version specific pages of the documentation
+2. Include links to change version (the version must be in the URL)
+3. Include the master version and last 3 minor releases. Documentation defaults to the last minor release.
+
+
+## Proposed solution
```

Comment:
Makes sense... but how? :smile:

Reply:
The goal of this proposal is primarily to get consensus about which pages should be versioned and how it should from the user-perspective. We decided that implementation details will be discussed deeper during the implementation phase (so that me and Jay can get a faster feedback loop, once we got consensus about versioning doc).

Reply:
@bwplotka Currently we use Docsy theme of HUGO. It allows versioning. https://www.docsy.dev/docs/adding-content/versioning/#adding-a-version-drop-down-menu 
This is the gist if where we can start with. 
https://www.docsy.dev/docs/adding-content/versioning/#adding-a-version-drop-down-menu